reference/sql/data-types/date-and-time.md

```diff
@@ -33,11 +303,11 @@ TIME[(fsp)]
 
 > **注意：**
 >
-> 注意 `TIME` 的缩写形式。例如，'11:12' 表示 '11:12:00' 而不是 '00:11:12'。但是，'1112' 表示 '00:11:12'。这些差异取决于 `:` 字符的存在与否。
+> 注意 `TIME` 的缩写形式。例如，`11:12` 表示 `11:12:00` 而不是 `00:11:12`。但是，`1112` 表示 `00:11:12`。这些差异取决于 `:` 字符的存在与否。
 
 ### `DATETIME` 类型
 
-`DATETIME` 类型是日期和时间的组合，格式为 `YYYY-MM-DD HH:MM:SS[.fraction]`。支持的范围是 `1000-01-01 00:00:00.000000` 到 `9999-12-31 23:59:59.000000`。`fsp` 参数表示秒精度，取值范围为 0~6，默认值为 0。
+`DATETIME` 类型是日期和时间的组合，格式为 `YYYY-MM-DD HH:MM:SS[.fraction]`。支持的范围是 `1000-01-01 00:00:00.000000` 到 `9999-12-31 23:59:59.999999`。`fsp` 参数表示秒精度，取值范围为 0~6，默认值为 0。TiDB 支持字符串或数字转换为 `DATETIME` 类型。
```

Comment:
@toutdesuite This section is a little bit different from the English version
![image](https://user-images.githubusercontent.com/41534398/81529862-4fb52000-9392-11ea-9d94-4dba453900e7.png) PTAL

Reply:
@qw4990 Please confirm whether the valid values range is to `9999-12-31 23:59:59.000000` or to `9999-12-31 23:59:59.999999`. Thanks!

Reply:
After testing, the actual maximum value of both `MySQL` and `TiDB` is `9999-12-31 23:59:59.499999`, but I think this is a bug.
According to [MySQL’s document](https://dev.mysql.com/doc/refman/8.0/en/datetime.html), the maximum value should be `9999-12-31 23:59:59.999999`, so I prefer to use `9999-12-31 23:59:59.999999` here.